deepmd/pt/model/atomic_model/linear_atomic_model.py

```
@@ -306,6 +308,66 @@ def forward_atomic(
+        mapping: torch.Tensor | None = None,
+        fparam: torch.Tensor | None = None,
+        aparam: torch.Tensor | None = None,
+        charge_spin: torch.Tensor | None = None,
+    ) -> dict[str, torch.Tensor]:
+        """Return the embedding of the first descriptor-fitting sub-model.
+
+        A linear/ZBL combination carries one learned descriptor-fitting model
+        (e.g. the DP part of a DP+ZBL model) whose embedding is well defined; the
+        per-sub-model neighbor lists are rebuilt exactly as in `forward_atomic`.
+        """
+        nframes, nloc, nnei = nlist.shape
```

Ruff (0.15.18)

[warning] Unpacked variable `nloc` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)